team-rotation.md

```diff
@@ -0,0 +1,18 @@
+# Team rotation
+
+The initial moderation team membership was defined through [RFC 102](https://github.com/NixOS/rfcs/blob/master/rfcs/0102-moderation-team.md#examples-and-interactions) and changes to the team are decided through a consensus among current moderation team members.
+
+## Joining the moderation team
+
+We are regularly on the lookout for new members, who whe think display good social skills, while interacting on the Discourse forum
```

Comment:
```suggestion
We are regularly on the lookout for new members, who whe think display good social skills, while interacting on GitHub, on the Discourse forum
```

Maybe? Although GitHub produces a lot less need for moderation.

Reply:
I think the point was more to describe where we usually source the data points from, not make it exclusive to those places. I still want to know about a God moderator even if they are on Reddit or other places.

Reply:
```suggestion
We are regularly on the lookout for new members, who we think display good social skills, while interacting on the Discourse forum
```

Reply:
> We are regularly on the lookout for new members, who we think display good social skills, while interacting on the community platforms.

Rewritten to be more generic.